docs/user-manual/address-settings.adoc

```diff
@@ -394,6 +394,11 @@ that helps to detect and prevent the processing of duplicate messages based on t
 By default, the `id-cache-size` setting inherits from the global `id-cache-size`, with a default of `20000` elements if not explicitly configured.
 Read more about xref:duplicate-detection.adoc#configuring-the-duplicate-id-cache[duplicate id cache sizes].
 
+initial-queue-buffer-size::
```

Comment:
There should be an entry for `initial-queue-bufffer-size` in the example `address-setting` above. The value used in the example should be the default.

Reply:
I made this change myself during the merge.